browser/brave_rewards/android/brave_rewards_native_worker.cc

```diff
@@ -301,24 +301,26 @@ void BraveRewardsNativeWorker::OnIsWalletCreated(bool created) {
         weak_java_brave_rewards_native_worker_.get(env), created);
 }
 
-void BraveRewardsNativeWorker::GetCurrentBalanceReport(JNIEnv* env,
+void BraveRewardsNativeWorker::GetBalanceReport(JNIEnv* env,
         const base::android::JavaParamRef<jobject>& obj) {
   if (brave_rewards_service_) {
-    return brave_rewards_service_->GetCurrentBalanceReport();
+    // TODO we need to get month and year from the UI
```

Comment:
@SergeyZhukovsky could you please update this so that you would pass current month and year in? Thank you